docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -127,13 +129,64 @@ performed, and instead Trino performs the aggregate processing.
                regionkey := tpch:regionkey
 
 Limitations
------------
+^^^^^^^^^^^
 
-Pushdown does not support a number of more complex statements:
+Aggregation pushdown does not support a number of more complex statements:
 
 * complex grouping operations such as ``ROLLUP``, ``CUBE``, or ``GROUPING SETS``
 * expressions inside the aggregation function call: ``sum(a * b)``
 * coercions: ``sum(integer_column)``
 * :ref:`aggregations with ordering <aggregate-function-ordering-during-aggregation>`
 * :ref:`aggregations with filter <aggregate-function-filtering-during-aggregation>`
 
+.. _join-pushdown:
+
+Join pushdown
+-------------
+
+Join pushdown allows the connector to delegate the table join operation to the
+underlying data source. This can result in performance gains, and allows Trino
+to perform the remaining query processing on a smaller amount of data.
+
+The specifics for the supported pushdown of table joins varies for each data
+source, and therefore for each connector.
+
+.. _limit-pushdown:
+
+Limit pushdown
+^^^^^^^^^^^^^^
+
+A :ref:`LIMIT or FETCH FIRST clause <limit-clause>` reduces the number of
+returned records for a statement. Limit pushdown enables a connector to push
+processing of such queries of unsorted record to the underlying data source.
+
+A pushdown of this clause can improve the performance of the query and
+significantly reduce the amount of data transferred from the data source to
+Trino.
+
+Queries include sections such as ``LIMIT N`` or ``FETCH FIRST N ROWS``.
+
+Implementation and support is connector-specific since different data sources
+support different SQL syntax.
```

Comment:
"different SQL syntax" is strange here, since a connector might be for a non-SQL data source. How about

> Implementation and support is connector-specific since different data sources have varying capabilities.

Reply:
Done